lib/chef/provider/package/macports.rb

```diff
@@ -3,7 +3,7 @@ class Provider
     class Package
       class Macports < Chef::Provider::Package
 
-        provides :macports_package, os: "mac_os_x"
+        provides :macports_package, os: "darwin"
```

Comment:
Yeah, but does it make any sense on linux/solaris/freebsd/etc?  Ultimately, it'd be nice to unload these classes that didn't apply to the o/s to reduce memory pressure.